src/common/api/hive.ts

```diff
@@ -11,7 +11,7 @@ import { vestsToRshares } from "../helper/vesting";
 import isCommunity from "../helper/is-community";
 
 import SERVERS from "../constants/servers.json";
-import { dataLimit } from "./bridge";
+import { dataLimit, getPost as getPostNew } from "./bridge";
```

Comment:
I think its better to rename the alias getPostNew does not seem intuitive. It tricks us into assuming that there may be a new method for fetching posts. May be give a new intuitive name like getPostsForTrail or getPostsForAccHistory :)